low-level/files/opt/seagate/sspl/setup/sspl_post_install.py

```diff
@@ -51,16 +53,17 @@ def __init__(self, args: str):
 
     def process(self):
         """Configure SSPL logs and service based on config."""
-        PRODUCT_NAME = Conf.get('global_config', 'release>product')
+        PRODUCT_NAME = Conf.get(GLOBAL_CONFIG, 'release>product')
 
         # Copy and load product specific sspl config
         if not os.path.exists(file_store_config_path):
             shutil.copyfile("%s/conf/sspl.conf.%s.yaml" % (SSPL_BASE_DIR, PRODUCT_NAME),
                             file_store_config_path)
 
-        # Global config path in sspl.conf will be referred by sspl-ll service later.
+        # Global config copy path in sspl.conf will be referred by sspl-ll service later.
         Conf.load("sspl", sspl_config_path)
         Conf.set("sspl", "SYSTEM_INFORMATION>global_config_url", self.global_config)
+        Conf.set("sspl", "SYSTEM_INFORMATION>global_config_dump_url", self.global_config_dump)
```

Comment:
`global_config_dump_url` , `global_config_url` why two config is needed ? We will be just referring dumped configs. So, sspl config should just have newly dumped config path

Reply:
This is just for SSPL reference where we dumped values from.
This could be useful in long running system if any config issue due to backend change and still we refer dumped config.